### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by brxzii** [minecraft-linux/mcpelauncher-manifest#826]
Any idea when the new update will be released? I can't join my friends realm

**Comment by Jepl4r** [comment]
Open the launcher, go in the settings and click on Open GameData Folder. Go in the profiles folder, open the profiles.ini file and change version=lock 971907302 to version=lock 971908101
(If you're using the x86 64 version change the first two numbers to 98)
After that, restart the launcher

**Comment by brxzii** [comment]
> Open the launcher, go in the settings and click on Open GameData Folder. Go in the profiles folder, open the profiles.ini file and change version=lock 971907302 to version=lock 971908101
> (If you're using the x86 64 version change the first two numbers to 98)
> After that, restart the launcher

What do I open it with? it says I can't open the profiles.ini file bc I have nothing to open it with

**Comment by brxzii** [comment]
> Open the launcher, go in the settings and click on Open GameData Folder. Go in the profiles folder, open the profiles.ini file and change version=lock 971907302 to version=lock 971908101
> (If you're using the x86 64 version change the first two numbers to 98)
> After that, restart the launcher

I think I figured it out but when I go to click play it says UNSUPPORTED VERSION Minecraft 1.19.81.01 perhaps I did something wrong?

**Comment by callmepizza45** [comment]
i had the same issues i fix it by leaving to minecraft beta beta program then it fixed it

**Comment by brxzii** [comment]
> i had the same issues i fix it by leaving to minecraft beta beta program then it fixed it

Im confused lol

**Comment by Jepl4r** [comment]
> I think I figured it out but when I go to click play it says UNSUPPORTED VERSION Minecraft 1.19.81.01 perhaps I did something wrong?

Enable Show unverified versions and Show incompatible versions from the Dev tab in the launcher settings.

**Comment by brxzii** [comment]
> > I think I figured it out but when I go to click play it says UNSUPPORTED VERSION Minecraft 1.19.81.01 perhaps I did something wrong?
> 
> Enable Show unverified versions and Show incompatible versions from the Dev tab in the launcher settings.

HI! When I do this and click play, the game doesn't load up. And when I go to click on minecraft it doesn't open so I have to restart my laptop inorder to open mc again.

**Comment by Jepl4r** [comment]
I don’t know. What happens if you leave only one of the two options enabled?

**Comment by brxzii** [comment]
> I don’t know. What happens if you leave only one of the two options enabled?

sadly it still didn't work. what app did you use to edit the text when you said to "change it to version=lock 971908101"

**Comment by Jepl4r** [comment]
A normal text editor. Anyway what architecture is the PC? x86_64 or Arm

**Comment by brxzii** [comment]
Its x86_64

**Comment by brxzii** [comment]
> A normal text editor. Anyway what architecture is the PC? x86_64 or Arm

Its x86_64

**Comment by Jepl4r** [comment]
Then change the first two numbers from 97 to 98 as I said in the first reply.

**Comment by brxzii** [comment]
Yeah I saw that part. I did change it, and it still didn't work. Not sure what the problem is since I did everything you said.

**Comment by Jepl4r** [comment]
I have no idea then, sorry.

**Comment by brxzii** [comment]
No worries, thank you though!
